src/store/filters/Sagas.ts

```diff
@@ -1,63 +1,44 @@
 import { takeEvery, all, call, select, put } from "redux-saga/effects";
-import { createNewFilter, FilterDimension } from "../../interfaces/IFilter";
+import { createNewFilter } from "../../interfaces/IFilter";
 import { IProjectInfo } from "../../interfaces/IProjectInfo";
+import { ProjectProperties } from "../../interfaces/IProjectInfo";
+import { ProjectFilterProperties } from "../../interfaces/IProjectInfo";
 import { ProjectsActionTypes } from "../projects/Types";
 import * as projectSelectors from "../projects/Selectors";
 import * as actions from "./Actions";
 
 const DEFAULT_FULL_TEXT_TITLE_VALUE = "";
 
-export function calculateTaskFilters(projects: IProjectInfo[]) {
+function calculateFilterStrings(projects: IProjectInfo[], propertyName : string) {
```

Comment:
NIT: There's a formatting error here. If you haven't already, please install a prettier plugin into your editor so we can keep the formatting consistent. If you're using IntelliJ or Webstorm, [there is one built in](https://www.jetbrains.com/help/webstorm/prettier.html) and [VSCode has a good one too](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).